plugins/html/help.md

```diff
@@ -1,60 +1,54 @@
 # Description
 
-Hypertext Markup Language (HTML) is the standard markup language for documents designed to be displayed in a web browser. This plugin provides the ability to convert an HTML document into a variety of formats using [pypandoc](https://pypi.python.org/pypi/pypandoc). Supported formats are:
-
-* DOCX
-* EPUB
-* Markdown
-* PDF
-* HTML5
-* Plain Text
+Hypertext Markup Language (HTML) is the standard markup language for documents designed to be displayed in a web browser. This plugin provides the ability to convert an HTML document into a variety of formats using [pypandoc](https://pypi.python.org/pypi/pypandoc). Supported formats are: DOCX, EPUB, Markdown, PDF, HTML5, Plain Text
```

Comment:
Feel free to change back the bullet list here if you'd like once you've finalised all the other changes 😄